### ISSUE TITLE
Syntax highlighting breaks on partials typed without trailing space

### ISSUE THREAD

**Issue by rosshamish** [zanuka/dust-buster#29]
Problem behaviour:
![2015-01-19 17_39_10-sametime appshare highlighter](https://cloud.githubusercontent.com/assets/1522678/5809055/4127b3d8-a002-11e4-8037-2958db87510c.png)

Expected behaviour:
![2015-01-19 17_40_04-sametime appshare highlighter](https://cloud.githubusercontent.com/assets/1522678/5809060/4b7bbb5e-a002-11e4-9d73-62b6c5896699.png)

Looks like the issue is in one of the regexes around [line 96 in syntaxes/DustBuster.YAML-tmLanguage](https://github.com/zanuka/dust-buster/blob/master/syntaxes/DustBuster.YAML-tmLanguage#L96)

``` yaml
dust_partials:
  begin: (\{*(>))\s*([-a-zA-Z0-9_\./]+)*
  beginCaptures:
    '1': {name: entity.name.tag.block.open.dust}
    '2': {name: constant.character.dust}
    '2': {name: constant.character.dust}
  end: (~?\/\})
  endCaptures:
    '1': {name: entity.name.tag.block.open.dust}
    '2': {name: variable.parameter.dust}
  name: meta.function.block.start.dust
  patterns:
  - {include: '#dust_strings'}
```

Dustjs documentation uses syntax without the space
![2015-01-19 17_44_25-sametime appshare highlighter](https://cloud.githubusercontent.com/assets/1522678/5809111/d8709eda-a002-11e4-95ab-643038f40a95.png)

**Comment by zanuka** [comment]
thanks @rosshamish , will fix

**Comment by rosshamish** [comment]
Updated this morning and the fix works for me. Awesome! 

For later reference: fixed in 394cd0677b75a42469f105d6c4965e25c5afed63

**Comment by zanuka** [comment]
good to hear :+1:
